protected/views/csv/upload.php

```diff
@@ -17,7 +10,7 @@
 );
 
 echo $form->fileField(new Csv(), 'csvFile');
-if(isset($errors) and $errors !== null) {
+if (isset($errors) and $errors !== null) {
     echo '<pre>';
     var_dump($errors);
     echo '</pre>';
```

Comment:
I noticed that it is possible to navigate to the preview screen when no file has been selected in the file upload field. I would assume that you should instead have validation errors. Make the field mandatory.